.specify/FEATURE_ROADMAP.md

```
@@ -0,0 +1,155 @@
+# WV Wild Outdoors - Feature Roadmap
+
+**Spec-Driven Development Sequence**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)